File: templates/matlab/chemistry_spectra_scenario_facets.m
function fig = chemistry_spectra_scenario_facets()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('small_multiples', 1909, 'chemistry spectra: scenario small multiples', 'chemistry spectra', 'scenario small multiples');
end

File: templates/python/chemistry_spectra_scenario_facets.py
"""chemistry_spectra_scenario_facets: 化学谱图场景分面（small-multiples 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='chemistry spectra: scenario small multiples'):
    return make_template_figure('small_multiples', seed=1909, title=title, domain='chemistry spectra', topic='scenario small multiples')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
